frontera/core/models.py

```diff
@@ -111,6 +140,27 @@ def url(self):
         return self._url
 
     @property
+    def url_as_unicode(self):
+        """
+        Returns the URL as unicode.
+        """
+        return to_unicode(self.url, self.encoding)
+
+    @property
+    def url_as_bytes(self):
+        """
+        Returns the URL as bytes.
+        """
+        return to_bytes(self.url, self.encoding)
+
+    @property
+    def encoding(self):
```

Comment:
I would remove encoding from these objects, please read this comment https://github.com/scrapinghub/frontera/pull/168#discussion_r72272128